namespace-to-cjs.rb

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
```

Comment:
Yep, like I mentioned it deserves a proper repo so we can start testing it and expanding upon it.